src/common/plugin/makeMetadata.ts

```diff
@@ -78,7 +78,7 @@ const fetchMetadata = async (memlet: Memlet): Promise<LocalWalletMetadata> => {
     const dataStr = await memlet.getJson(metadataPath)
     return JSON.parse(dataStr)
   } catch {
-    return await resetMetadata(memlet)
+    return asLocalWalletMetadataCleaner(await resetMetadata(memlet))
```

Comment:
You don't need to clean "internal" functions that returns a known type.
Cleaners are used when interacting with external data like other servers or databases.